packages/compressed-textures/src/loaders/DDSLoader.ts

```diff
@@ -63,7 +63,8 @@ const DDS_DX10_FIELDS = {
  */
 // This is way over-blown for us! Lend us a hand, and remove the ones that aren't used (but set the remaining
 // ones to their correct value)
-enum DXGI_FORMAT {
+enum DXGI_FORMAT
+    {
```

Comment:
Why are enums getting this extra indent on the next line?

Reply:
Seems like a known bug https://github.com/typescript-eslint/typescript-eslint/issues/3225